ydb/library/yql/minikql/comp_nodes/mkql_grace_join_imp.cpp

```diff
@@ -55,11 +58,15 @@ void TTable::AddTuple(  ui64 * intColumns, char ** stringColumns, ui32 * strings
         char * currStrPtr = reinterpret_cast< char* > (startPtr);
 
         for( ui64 i = 0; i < NumberOfKeyStringColumns; i++) {
+            WriteUnaligned<ui32>(currStrPtr, stringsSizes[i] );
```

Comment:
nit: It would be nice to change `sizeof(ui32)` to `sizeof(<variable name>)` here and in other places. In the future, changing the type of a variable may lead to errors that are difficult to detect

Reply:
Hmm. Given that this is serialization that must match on pack/unpack side, that can be some `TStringOffset`. Using type of variable feels equally fragile (e.g. `auto variable = foo.size()` would be sleeping disaster).